.github/workflows/ci.yml

```diff
@@ -0,0 +1,111 @@
+name: CI
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  test-module:
+    name: Test module builds
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v1
+      with:
+       node-version: '12.x'
+    - name: Install Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
```

Comment:
Do we require 3.7? 😟 It's [EOL](https://endoflife.date/python) in June 2023

Reply:
No, I just copied this from somewhere. Once merged, I'll fix this action up.